addons/mail/static/src/models/activity/activity.js

```diff
@@ -283,6 +283,13 @@ function factory(dependencies) {
         creator: many2one('mail.user'),
         dateCreate: attr(),
         dateDeadline: attr(),
+        /**
+         * Hold the feedback text for the activity popover. This only to restore
+         * the textarea inside the popover and cannot be trusted
+         */
```

Comment:
1st part of comment is good. The 2nd part is not super clear, could be better worded to highlight this field has restricted application. Suggestion:
```js
/**
 * Backup of the feedback content of an activity to be marked as done.
 * Feature-specific to restoring the feedback content when component is re-mounted.
 * In all other cases, this field value should not be trusted.
 */
```